iree/compiler/Conversion/CodegenUtils/WorkgroupCalculation.cpp

```diff
@@ -0,0 +1,210 @@
+#include "iree/compiler/Dialect/HAL/IR/HALOps.h"
+#include "iree/compiler/Dialect/HAL/Utils/TypeUtils.h"
+#include "iree/compiler/Dialect/Shape/IR/ShapeOps.h"
+#include "llvm/ADT/SetVector.h"
```

Comment:
Could we rename this to GetNumWorkgroups.cpp

Reply:
Done!